docs/src/main/asciidoc/chapters/administration.txt

```diff
@@ -404,6 +399,51 @@ and BatchScanner passing in the name of the context, app1 in the example above.
 iterators to load classes from the locations defined by the context. Passing the context name to the scanners allows you to override the table setting
 to load only scan time iterators from a different location. 
 
+=== Vendor-specific configuration
```

Comment:
I pushed another commit that removes the vendor-specific documentation from the user manual.